core-api-library/library/src/main/java/net/gini/android/core/api/RetrofitHelper.kt

```diff
@@ -0,0 +1,52 @@
+package net.gini.android.core.api
+
+import net.gini.android.core.api.authorization.UserService
+import okhttp3.Interceptor
+import okhttp3.OkHttpClient
+import okhttp3.Request
+import retrofit2.Retrofit
+import retrofit2.converter.gson.GsonConverterFactory
+import java.util.concurrent.TimeUnit
+
+object RetrofitHelper {
```

Comment:
The logic in this helper will have to be moved to the GiniCoreAPIBuilder. That class is responsible for configuring, instantiating and wiring up all needed classes.